### PR TITLE
[TASK] Implement version system

### PR DESCRIPTION
**Pull request by SandroCoutinho**
Related to issue #10.

### CODE REVIEW

**Comment by JKetelaar** [comment]
Thanks, we'll need to update issue #10 with the nightly problem.
Though nightly versions aren't possible with this yet, so no worries.

Looking good, thanks!

File: src/main/java/org/parabot/launcher/data/Configuration.java [hunk 1]
```diff
@@ -12,5 +12,5 @@
     public static final String CONFIG_NAME = "launcher";
     public static final String LAUNCHER_CONFIG_LOCATION = Directories.getSettingsPath() + File.separator + Configuration.CONFIG_NAME + ".json";
     public static final String CLIENT_LOCATION = Directories.getCachePath() + File.separator + BOT_TITLE + ".jar";
-    public static final String GET_BOT_VERSION = "http://bdn.parabot.org/api/v2/bot/version";
+    public static final String GET_BOT_VERSION = "http://v3.bdn.parabot.org/api/bot/list/client?limit=1";
```
Comment (JKetelaar) [inline src/main/java/org/parabot/launcher/data/Configuration.java]: Doesn't the current bot use this check too?

File: src/test/java/org/parabot/launcher/VersionTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import org.parabot.launcher.helpers.VersionHelper;
+
+/**
+ * @author EmmaStone
+ */
+public class VersionTest {
+
+    @Test
+    public void testAmount() {
+        VersionHelper.getLatestClient();
```
Comment (JKetelaar) [inline src/test/java/org/parabot/launcher/VersionTest.java]: Do something like `Assert.isNotNull`

File: src/main/java/org/parabot/launcher/data/Configuration.java (hunk 1 above)
Comment (SandroCoutinho) [reply src/main/java/org/parabot/launcher/data/Configuration.java]: No it doesn't maybe that's something to be added in the future.

Comment (JKetelaar) [reply src/main/java/org/parabot/launcher/data/Configuration.java]: I'm saying, that's using the same constant name; https://github.com/Parabot/Parabot/blob/development/src/main/java/org/parabot/core/Configuration.java#L18

**Pull request merged**